src/components/RestaurantHeader/RestaurantHeader.tsx

```diff
@@ -10,67 +14,61 @@ type Props = {
 }
 
 const RestaurantHeader: React.FC<Props> = ({ restaurant }) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
   return (
-    <Box style={styles.restaurantHeader}>
+    <Box>
       <ImageBackground
-        style={styles.hero}
-        source={{
-          uri: `${assetsUrl}/${restaurant?.images.banner}`,
-        }}
+        style={styles.heroBackground}
+        source={{ uri: `${assetsUrl}/${restaurant?.images.banner}` }}
       >
-        <Box padding="m" style={styles.heroTextContainer}>
+        <Box padding={["xs", "m"]} margin={["s", "none"]} style={styles.hero}>
```

Comment:
I thought about implementing `paddingVertical` and `paddingHorizontal` as RN does, but I like this format better.

Reply:
It mirrors how css does it. Theoreticalyl it could be expanded to the 3 and 4 term variants ,but I didn't need it.